yarn.lock

```diff
@@ -1941,12 +3108,17 @@ caniuse-lite@^1.0.30001219:
   resolved "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001242.tgz#04201627abcd60dc89211f22cbe2347306cda46b"
   integrity sha512-KvNuZ/duufelMB3w2xtf9gEWCSxJwUgoxOx5b6ScLXC4kPc9xsczUVCPrQU26j5kOsHM4pSUL54tAZt5THQKug==
 
+caniuse-lite@^1.0.30001312:
```

Comment:
**Question** Why are there two caniuse-lite versions now? Is that an accident? (See line 3106)

Reply:
It is an accident caused due to to different versions of `browserlist`. Resolved it through this [8f7b6d7](https://github.com/signavio/i18n/pull/131/commits/8f7b6d71f8427579843aae11967953d564b632d8)